src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -52,25 +52,25 @@
       <DataSource Persistence="ProjectFileWithInterception" PersistedName="OutputTypeEx" HasConfigurationCondition="False" SourceOfDefaultValue="AfterContext" />
     </EnumProperty.DataSource>
   </EnumProperty>
-  <StringProperty Name="StartupObject" DisplayName="Type that contains the entry point" Visible="True"/>
-  <StringProperty Name="ApplicationManifest" DisplayName="Application Manifest" Visible="True">
+  <StringProperty Name="StartupObject" DisplayName="Type that contains the entry point" Visible="False"/>
+  <StringProperty Name="ApplicationManifest" DisplayName="Application Manifest" Visible="False">
     <StringProperty.DataSource>
       <DataSource Persistence="ProjectFileWithInterception" PersistedName="ApplicationManifest" HasConfigurationCondition="false" SourceOfDefaultValue="AfterContext" />
     </StringProperty.DataSource>
   </StringProperty>
-  <StringProperty Name="Win32ResourceFile" DisplayName="Win32 Resource File" Visible="True">
+  <StringProperty Name="Win32ResourceFile" DisplayName="Win32 Resource File" Visible="False">
     <StringProperty.DataSource>
       <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" HasConfigurationCondition="False" SourceOfDefaultValue="AfterContext" />
     </StringProperty.DataSource>
   </StringProperty>
-  <StringProperty Name="PreBuildEvent" DisplayName="Pre Build Event" Visible="True"/>
-  <StringProperty Name="PostBuildEvent" DisplayName="Post Build Event" Visible="True"/>
-  <EnumProperty Name="RunPostBuildEvent" DisplayName="Run Post Build Event" Visible="True">
+  <StringProperty Name="PreBuildEvent" DisplayName="Pre Build Event" Visible="False"/>
+  <StringProperty Name="PostBuildEvent" DisplayName="Post Build Event" Visible="False"/>
+  <EnumProperty Name="RunPostBuildEvent" DisplayName="Run Post Build Event" Visible="False">
     <EnumValue Name="Always" DisplayName="Always" />
     <EnumValue Name="OnBuildSuccess" DisplayName="On successful build"  IsDefault="True" />
     <EnumValue Name="OnOutputUpdated" DisplayName="When the build updates the project output" />
   </EnumProperty>
-  <StringProperty Name="ReferencePath" DisplayName="Reference Path" Visible="True"/>
+  <StringProperty Name="ReferencePath" DisplayName="Reference Path" Visible="False"/>
   <StringProperty Name="FileName" DisplayName="Project File" ReadOnly="True">
```

Comment:
Yes leaving these two as visible makes sense - since that matches the legacy project system.